common/test/common_tests/types/text_test.cljc

```diff
@@ -0,0 +1,277 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think there should also be a test at the `common.logic` level, to check the whole cycle of creating a text shape in a component, changing something in a copy and synchronizing it.

The test does not need to address all possible cases (since they are tested in those ones in `common.types`), but at least ensure that the helper functions are called.